Assets/Scripts/Controllers/PlayerController.cs

```diff
@@ -202,6 +202,7 @@ public void OnTriggerExit(Collider collision)
     protected override void Die()
     {
         Debug.Log("PLAYER HAS DIED");
+        goldIndicator.DecrementGold(5, transform.position);
```

Comment:
gold should not be decremented upon player death, only upon purchases from the merchant, when that gets implemented